enrolments/admin.py

```diff
@@ -12,4 +13,13 @@ class ClassAdmin(admin.ModelAdmin):
     search_fields = ("name",)
 
 
+class EnrolmentAdmin(admin.ModelAdmin):
+    list_display = ("id", "family", "created_at", "updated_at", "enrolled_class")
+
+    ordering = ("-id",)
+
+    search_fields = ("family__id", "family__email", "enrolled_class___name")
```

Comment:
one excess underscore in `"enrolled_class___name"` but otherwise. LGTM!

Reply:
Great catch! Fixed with the new commit.